cpp/memilio/config.h

```diff
@@ -67,6 +68,15 @@ struct Limits<double> {
         return 1e-12;
     }
 };
+
+template <class AD_TAPE_REAL, class DATA_HANDLER>
+struct Limits<ad::internal::active_type<AD_TAPE_REAL, DATA_HANDLER>> {
+    /// @brief Returns the limit under which an ad::internal::active_type may be rounded down to zero.
```

Comment:
I think using "AD values" or "AD value_type" instead of the internal name is more helpful in the documentation.

I am also a bit torn on the template names. As is, it is easy to find their original use and/or documentation, but they technically do not conform to our coding guidelines. I slightly lean to the conforming option, maybe use "AdReal" (or "AdTapeReal") and "DataHandler"?

Reply:
Yes ok, I will adapt this to be in accordance with our guidelines.